images/heapster/Makefile

```diff
@@ -5,18 +5,21 @@ VER=v1.0.2
 PREFIX=gravity
 GOFLAGS=-a -installsuffix cgo --ldflags '-w'
 OUT=/targetdir/heapster
+BUILD_UID?=1000
 
 all: $(OUT)
 
 $(OUT):
+	useradd --uid $$BUILD_UID builder
```

Comment:
Because we should create user on every debian-venti usage with different UID. Like on my PC I have 1000, and on some testing box we have 1001.